api/v1/flow/flow.proto

```diff
@@ -350,6 +350,7 @@ enum DropReason {
     INVALID_TC_BUFFER = 184;
     NO_SID = 185;
     MISSING_SRV6_STATE = 186;
+    DROP_UNENCRYPTED_TRAFFIC = 189;
```

Comment:
```suggestion
    UNENCRYPTED_TRAFFIC = 189;
```
For consistency with the other values in the `DropReason` enum, let's drop `DROP`.